moveit_plugins/moveit_ros_control_interface/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(moveit_ros_control_interface)
 
-add_compile_options(-std=c++11)
+if(NOT MSVC)
+  add_compile_options(-std=c++11)
+else()
+  add_compile_options(/Zc:__cplusplus)
+  add_definitions(-DNO_STRICT)
```

Comment:
We should no longer need NO_STRICT since we don't have Windows.h pollution problem. I will remove it.